src/srml/system.rs

```diff
@@ -49,7 +50,8 @@ pub trait System: 'static + Eq + Clone + std::fmt::Debug {
     /// The block number type used by the runtime.
     type BlockNumber: Parameter
         + Member
-        + MaybeSerializeDebug
+        + MaybeSerializeDeserialize
```

Comment:
Deserialize has been bound now because some requirement has changed on blocknumber as well, it should be a problem.